File: Pemro1_Modul4/Python/PRAK404-2510817320010-PutriDeissytaHerliyana.py
while True:
    print("Pilih program")
    print("1. Penjumlahan")
    print("2. Pengurangan")
    print("3. Perkalian")
    print("4. Pembagian")
    print("5. Exit")

    pilihan = int(input("Masukkan Pilihan : "))

    if pilihan == 5:
        print("Terimakasih, telah menggunakan kalkulator PUTRI DEISSYTA HERLIYANA")
        break
    elif pilihan >= 1 and pilihan <= 4:
        nilai1 = float(input("Masukkan nilai pertama : "))
        nilai2 = float(input("Masukkan nilai kedua : "))

        if pilihan == 1:
            hasil = nilai1 + nilai2
            operasi = "Penjumlahan"
        elif pilihan == 2:
            hasil = nilai1 - nilai2
            operasi = "Pengurangan"
        elif pilihan == 3:
            hasil = nilai1 * nilai2
            operasi = "Perkalian"
        elif pilihan == 4:
            if nilai2 == 0:
                print("Error: Pembagian dengan nol tidak diperbolehkan.")
                continue
            hasil = nilai1 / nilai2
            operasi = "Pembagian"

        print(f"Hasil {operasi} antara {nilai1:.2f} dengan {nilai2:.2f} adalah {hasil:.2f}")
        print()
    else:
        print("Input anda salah, silahkan coba lagi\n")

File: Pemro1_modul4/C/PRAK404-2510817320010-PutriDeissytaHerliyana.c
#include <stdio.h>

int main() {
    int pilihan;
    float nilai1, nilai2, hasil;

    while (1) {
        printf("Pilih program\n");
        printf("1. Penjumlahan\n");
        printf("2. Pengurangan\n");
        printf("3. Perkalian\n");
        printf("4. Pembagian\n");
        printf("5. Exit\n");
        printf("Masukkan Pilihan : ");
        scanf("%d", &pilihan);

        if (pilihan == 5) {
            printf("Terimakasih, telah menggunakan kalkulator PUTRI DEISSYTA HERLIYANA\n");
            break;
        } else if (pilihan >= 1 && pilihan <= 4) {
            printf("Masukkan nilai pertama : ");
            scanf("%f", &nilai1);
            printf("Masukkan nilai kedua : ");
            scanf("%f", &nilai2);

            if (pilihan == 1) {
                hasil = nilai1 + nilai2;
                printf("Hasil Penjumlahan antara %.2f dengan %.2f adalah %.2f\n\n", nilai1, nilai2, hasil);
            } else if (pilihan == 2) {
                hasil = nilai1 - nilai2;
                printf("Hasil Pengurangan antara %.2f dengan %.2f adalah %.2f\n\n", nilai1, nilai2, hasil);
            } else if (pilihan == 3) {
                hasil = nilai1 * nilai2;
                printf("Hasil Perkalian antara %.2f dengan %.2f adalah %.2f\n\n", nilai1, nilai2, hasil);
            } else if (pilihan == 4) {
                if (nilai2 == 0) {
                    printf("Error: Pembagian dengan nol tidak diperbolehkan.\n\n");
                    continue;
                }
                hasil = nilai1 / nilai2;
                printf("Hasil Pembagian antara %.2f dengan %.2f adalah %.2f\n\n", nilai1, nilai2, hasil);
            }
        } else {
            printf("Input anda salah, silahkan coba lagi\n\n");
        }
    }

    return 0;
}
